src/components/clusterConfiguration/DiscoveryTroubleshootingModal.tsx

```diff
@@ -50,78 +50,66 @@ export const DiscoveryTroubleshootingModal: React.FC<DiscoveryTroubleshootingMod
     >
       <TextContent>
         <Text component={TextVariants.p}>
-          Booting from an ISO takes a few minutes. It also depends heavily on the hardware and
-          network configuration. However, if you have verified that your host machine is powered on,
-          DHCP is enabled and yet the host does not appear in the inventory list, then login to the
-          host over SSH to troubleshoot. For that, you would need to know your machine IP-address.
-          You can also use a console such as BMC or virtual machine console.
+          To troubleshoot host discovery issues, complete the following steps:
         </Text>
+        <Text component={TextVariants.h2}>Verify that your host machine is powered on</Text>
+        <Text component={TextVariants.p}>
+          Note: Boot time depends on several factors such as your hardware and network
+          configuration, and if you are booting from an ISO.
+        </Text>
+        <Text component={TextVariants.h2}>Verify if DHCP is enabled</Text>
         <Text component={TextVariants.h2}>SSH into your machine</Text>
-        <PrismCode code={`ssh core@<machine-ip>`} />
         <Text component={TextVariants.p}>
-          The SSH key provided when generating the discovery ISO is used to authenticate the user.
-          It should not be required to provide the password.
```

Comment:
replace "It should not be required to provide the password." by "It is not mandatory to provide the password."

Reply:
I think you actually can't use a password to login in this case, only via ssh key.